NachoSpigot-Server/src/main/java/net/minecraft/server/EntityTracker.java

```diff
@@ -1,41 +1,44 @@
 package net.minecraft.server;
 
-import com.google.common.collect.Lists;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
 
-import java.util.*;
-import java.util.concurrent.Callable;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 import me.elier.nachospigot.config.NachoConfig;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
+import me.rastrian.dev.utils.IndexedLinkedHashSet;
 public class EntityTracker {
 
-    private static final Logger a = LogManager.getLogger();
-    private final WorldServer world;
-
-    public Set<EntityTrackerEntry> c = new io.papermc.paper.util.maplist.ObjectMapList<>(); // IonSpigot - HashSet -> ObjectMapList
-    public Set<EntityTrackerEntry> getTrackedEntities() { return c; }
+    public IndexedLinkedHashSet<EntityTrackerEntry> entitytTrackerHashset = new IndexedLinkedHashSet<>();
```

Comment:
Do **NOT** do this! Rename it back to `c` or you will break plugins such as Citizens! Use the obfhelper function below if you want!

Do not use this suggestion, since you need to do this from your IDE everywhere, but as example:

```suggestion
    public IndexedLinkedHashSet<EntityTrackerEntry> c = new IndexedLinkedHashSet<>();
```